stream_alert/shared/lookup_tables/driver_s3.py

```diff
@@ -61,7 +62,13 @@ def __init__(self, configuration):
 
         # Explicitly set timeout for S3 connection. The boto default timeout is 60 seconds.
         boto_config = boto_helpers.default_config(timeout=10)
-        self._s3_client = boto3.resource('s3', config=boto_config)
+
+        self._s3_adapter = S3Adapter(
+            self,
+            boto3.resource('s3', config=boto_config),
```

Comment:
see other comment in `S3Adapter` - but I _think_ you could just create the `Bucket` resource right here instead of passing in the `resource` and `bucket_name` as args here.. like:

```
self._s3_adapter = S3Adapter(
    self,
    boto3.resource('s3', config=boto_config).Bucket(self._s3_bucket),
    self._s3_key
)
```